src/backend-query.js

```diff
@@ -6,6 +6,9 @@ const metagraphUrl =
 // url for hetio definitions (metanodes, properties, etc)
 const hetioDefinitions =
   'https://raw.githubusercontent.com/hetio/hetionet/master/describe/definitions.json';
+// url for hetio styles (metanode fill/text color, etc)
+const hetioStyles =
+  'https://raw.githubusercontent.com/hetio/hetionet/master/describe/styles.json';
```

Comment:
should use versioned links here and above, so `hetio/hetionet` doesn't have to worry about breaking this page with changes. In general, we want all links to github assets to be versioned unless the repo has made a committment to always keep a file at a location

Reply:
Isn't one of the points of pulling the colors dynamically that we don't have to rebuild the whole front end if we update the colors? The way it is in this PR means that you could simply do a change in the hetio repo, and all copies of the hetmech front end will immediately update.

Reply:
The point is to prevent repetition of where the colors are defined. hetio/hetionet makes no guarantee about the file remaining in that location. In addition, hetio/hetionet could have a major update, such that the node types entirely change. Downstream applications should opt-in to updates. My experience with linking to assets in GitHub repos is that you're much better off versioning than taking the risks of the repo changing in some way you don't expect.

Reply:
Okay. Should I change all the urls in this PR or should it be a separate PR.

Reply:
I guess let's merge this with the new unversioned link and then a subsequent PR can change everything.